src/components/diagrams/diagram-common.ts

```diff
@@ -112,6 +112,9 @@ export const styles = {
         '& .nad-edge-infos': {
             opacity: NAD_INVALID_LOADFLOW_OPACITY,
         },
+        '& .sld-overload .sld-vl-overvoltage .sld-vl-undervoltage': {
```

Comment:
Move this line 112, to keep SLD CSS classes conf and NAD CSS classes conf together ?

Reply:
done